typings/index.d.ts

```diff
@@ -1387,6 +1387,7 @@ declare module 'discord.js' {
     public shards: Collection<number, Shard>;
     public token: string | null;
     public totalShards: number | 'auto';
+    public shardList: number | 'auto';
```

Comment:
shardList is an array of numbers or 'auto'
https://github.com/discordjs/discord.js/blob/80e1afb65447e1d66429ddda36902d871920fdc3/src/sharding/ShardingManager.js#L64-L82
https://github.com/discordjs/discord.js/blob/624a4464ca86bfa0b095ecb2cdaac2e8030cc413/typings/index.d.ts#L1373

Reply:
Updated to `number[]`